Marlin/src/HAL/STM32/Sd2Card_sdio_stm32duino.cpp

```diff
@@ -128,7 +128,7 @@
     Init.ClockPowerSave      = hsd.Init.ClockPowerSave;
     Init.BusWide             = hsd.Init.BusWide;
     Init.HardwareFlowControl = hsd.Init.HardwareFlowControl;
-    Init.ClockDiv            = sdio_set_clock(SDIO_CLOCK);
+    Init.ClockDiv            = clock2divider(SDIO_CLOCK);
```

Comment:
Is this referring to "clock2", or "clock to divider"?

If the second, please use words and not numbers. `clock_to_divider`